### DISCUSSION TITLE
Icon request: Bull

### DISCUSSION THREAD

**Discussion by ericbrown** [FortAwesome/Font-Awesome#21135] [Icon Requests]
I would like to request a new icon for use in all things financial market related. The icon, which is attached, is a Bull and  would be used to denote 'bullish' activity in the market.
![bull-icon](https://f.cloud.github.com/assets/449599/523837/5d46258a-c0e4-11e2-9cd2-db147b6d00ae.png)

**Comment by G-Driver** [comment]
+1

**Comment by jcham83** [comment]
+1

**Comment by Crissov** [comment]
- 🐂 [U+1F402](http://emojipedia.org/ox/) Ox / Bull Unicode emoji #222
- 🐃 [U+1F403](http://emojipedia.org/water-buffalo/) Aurox / Water Buffalo
- 🐄 [U+1F404](http://emojipedia.org/cow/) Cow → #4166 
- 🐮 [U+1F42E](http://emojipedia.org/cow-face/) Cow Face / Cattle / Beef
- ♉️ [U+2649](http://emojipedia.org/taurus/) Taurus (bull zodiac sign)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
